kadi_apps/blueprints/find_attitude/templates/find_attitude/index.html

```diff
@@ -38,10 +38,16 @@ <h1 align="center" style="margin: 0px; padding: 0px;"> Find Chandra attitude fro
   <button type="submit" name="action" value="gettelem">Get Telem</button>
   <div class="row">
     <div class="col">
+      <div class="row">
     <div class="col">
       <label for="distance_tolerance">Distance tolerance (arcsec) </label>
       <input type="text" class="form-control" id="distance_tolerance" name="distance_tolerance" value="{{distance_tolerance}}">
     </div>
+    <div class="col">
+      <label for="maude_channel">MAUDE channel</label>
```

Comment:
Although there are other legal values of MAUDE channel, I think providing a hint to the two that are likely to be used is useful.
```suggestion
      <label for="maude_channel">MAUDE channel (FLIGHT or ASVT)</label>
```

Reply:
I added the drop-down (which didn't work the first time I tried but seems to be working now).